src/cpp/rtps/participant/RTPSParticipantImpl.cpp

```diff
@@ -2539,6 +2539,8 @@ bool RTPSParticipantImpl::ignore_participant(
         {
             std::unique_lock<shared_mutex> _(ignored_mtx_);
             ignored_participants_.insert(participant_guid);
+            pdp()->remove_remote_participant(GUID_t(participant_guid, c_EntityId_RTPSParticipant),
```

Comment:
Should this be done with `ignored_mtx_` taken?

Reply:
Let's take it out of the mutex scope just in case.